app/src/main/java/com/github/studydistractor/sdp/ui/EventScreen.kt

```diff
@@ -0,0 +1,127 @@
+package com.github.studydistractor.sdp.ui
+
+import android.content.Context
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Chat
+import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.studydistractor.sdp.event.EventViewModel
+
+@Composable
+fun EventScreen(
+    eventViewModel : EventViewModel,
+    onOpenChatClick: () -> Unit,
+) {
+    fun showFailureToast(context: Context, message: String) {
+        Toast.makeText(context, "Failure: $message", Toast.LENGTH_SHORT)
+            .show()
+    }
+
+    val uiState by eventViewModel.uiState.collectAsState()
+    val context = LocalContext.current
+
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top
+    ) {
+        item {
+            Spacer(modifier = Modifier.height(16.dp))
```

Comment:
If the `Spacer` is for each item, maybe you can use some verticalArrangement method instead ?

Reply:
Very true, it will be much better